logstash-core-event-java/src/main/java/org/logstash/Event.java

```diff
@@ -338,22 +348,33 @@ private Timestamp initTimestamp(Object o) {
             logger.warn("Error parsing " + TIMESTAMP + " string value=" + o.toString());
         }
 
-        tag(TIMESTAMP_FAILURE_TAG);
-        this.data.put(TIMESTAMP_FAILURE_FIELD, o);
-
-        return Timestamp.now();
+        return null;
```

Comment:
woops, I'll re-read. tried to expand to the point that I thought I saw the state of the world, should have checked the raw and not the diff